sklearn/model_selection/tests/test_search.py

```diff
@@ -863,6 +870,16 @@ def test_random_search_cv_results():
 
     n_splits = 3
     n_search_iter = 30
+    params = dict(C=expon(scale=10), gamma=expon(scale=0.1))
```

Comment:
Why do you need to add these lines? Seems duplicate?
https://github.com/gxyd/scikit-learn/blob/890ce677065a4ef90a5c3771e096a5876c46856b/sklearn/model_selection/tests/test_search.py#L897-L900

Reply:
probably the same case here as well (not resolving the merge conflicts cleanly).